samples/Network/enforce-firewall-route-table-on-subnet/azurepolicy.json

```diff
@@ -0,0 +1,55 @@
+{
+	"type": "Microsoft.Authorization/policyDefinitions",
+	"name": "enforce-firewallroutetable-on-subnet",
+	"properties": {
+		"displayName": "Route table of every subnet",
+		"description": "This policy enforces a specific Route Table on every subnet",
+		"parameters": {
+			"rtId": {
+				"type": "String",
+				"metadata": {
+					"description": "Resource Id of the Firewall route table",
```

Comment:
Please add an example of what an expected parameter value would look like in the description